examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,487 @@
+
+mapdl.nsel("ALL")
+mapdl.finish()
+
+###############################################################################
+# Solve
+# ~~~~~
+# Enter solution mode and solve the system. Print the solver output.
+
+# Start solution presederu
+mapdl.run("/SOLU")
+
+# Define solution function.
+def solution(deflect=None):
+    mapdl.nsel("R", "LOC", "Z", 10)
+    mapdl.d(node="ALL", lab="UZ", value=deflect)
+    mapdl.nsel("ALL")
+    mapdl.solve()
+
+
+# Run each load step to reproduce needed deflection subsequently.
+# Load Step 1
+solution(deflect=defl_ls1)
+
+# Load Step 2
+solution(deflect=defl_ls2)
+
+# Load Step 3
+solution(deflect=defl_ls3)
+mapdl.finish()
+
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing.
+
+# Enter the post-processing routine
+mapdl.run("/OUT,")
+mapdl.post1()
+
+
+###############################################################################
+# Define loads.
+# ~~~~~~~~~~~~~
+# Set up the function.
+
+def getload():
+    ####
+    # Select the nodes in the PIPE288 element model.
+    mapdl.nsel("S", "NODE", "", 1, 2)
+    mapdl.nsel("R", "LOC", "Z", 0)
+    mapdl.run("/OUT,SCRATCH")
+
+    # Sum the nodal force contributions of elements.
+    mapdl.fsum()
+
+    # Extrapolation of the force results in the full 360 (deg) model.
+    load_288 = mapdl.get_value("FSUM", 0, "ITEM", "FZ")
+
+    ####
+    # Select the nodes in the SOLID185 element model.
+    mapdl.nsel("S", "NODE", "", 101, 118)
+    mapdl.nsel("R", "LOC", "Z", 0)
+    mapdl.fsum()
+
+    # Get the force value of the simplified model.
+    load_185_theta = mapdl.get_value("FSUM", 0, "ITEM", "FZ")
+
+    # Extrapolation of the force results in the full 360 (deg) model.
+    load_185 = load_185_theta * 360 / theta
+    mapdl.run("*STATUS,LOAD")
+
+    ####
+    # Select the nodes in the SHELL181 element model.
+    mapdl.nsel("S", "NODE", "", 201, 212)  # SELECT NODES IN SHELL181 MODEL
+    mapdl.nsel("R", "LOC", "Z", 0)
+
+    # Sum the nodal force contributions of elements.
+    mapdl.fsum()
+    mapdl.run("/OUT,")
+
+    # Get the force value of the simplified model.
+    load_181_theta = mapdl.get_value("FSUM", 0, "ITEM", "FZ")
+
+    # Extrapolation of the force results in the full 360 (deg) model.
+    load_181 = load_181_theta * 360 / theta
+    mapdl.run("*STATUS,LOAD")
+
+    # Return load results of each element model.
+    return load_288, load_185, load_181
+
+
+###############################################################################
+# Define loads for load step 1
+
+for i in range(1, ls + 1):
+    mapdl.set(1, 1)
+    load_288, load_185, load_181 = getload()
+    results_load_deflection = f"""
+    LOAD STEP {i}
+    PIPE288_LS{i}= {abs(round(load_288,0))}
+    SOLID185_LS{i} = {abs(round(load_185,0))}
+    SHELL181_LS{i} = {abs(round(load_181,0))}
+    
+    """
+    print(results_load_deflection)
```

Comment:
There should be less execution within f-strings, only the final values should be evaluated as this is far easier to debug.